.github/workflows/build.yml

```diff
@@ -73,9 +73,11 @@ jobs:
               -DMAD=ON
               -DMEDIAFOUNDATION=ON
               -DSTATIC_DEPS=ON
+              -DCMAKE_VERBOSE_MAKEFILE=ON
             cc: cl
             cxx: cl
-            cmake_generator: Ninja
+            # cmake_generator: Ninja
+            cmake_generator: Visual Studio 16 2019
```

Comment:
Why did you change this? This is off topic from this PR.

Reply:
To see the verbose output. 

Still see 
```
shout_mixxx.lib(thread.obj) : error LNK2001: unresolved external symbol __imp_pthread_attr_setinheritsched [D:\a\mixxx\mixxx\build\mixxx-test.vcxproj]
```

Even though pthread.lib is included in the linker command

```
C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\VC\Tools\MSVC\14.28.29333\bin\HostX64\x64\link.exe /ERRORREPORT:QUEUE /OUT:"D:\a\mixxx\mixxx\build\Debug\mixxx.exe" 
...
"C:\buildenv\2.3-j00019-x64-release-fastbuild-static-55e94982-minimal\lib\rubberband.lib" "C:\buildenv\2.3-j00019-x64-release-fastbuild-static-55e94982-minimal\lib\libsndfile.lib" "C:\buildenv\2.3-j00019-x64-release-fastbuild-static-55e94982-minimal\lib\g72x.lib" Debug\SoundTouch.lib "C:\buildenv\2.3-j00019-x64-release-fastbuild-static-55e94982-minimal\lib\tag.lib" "C:\buildenv\2.3-j00019-x64-release-fastbuild-static-55e94982-minimal\lib\libHSS1394.lib" "lib\libshout-idjc\Debug\shout_mixxx.lib" "C:\buildenv\2.3-j00019-x64-release-fastbuild-static-55e94982-minimal\lib\pthread.lib" "C:\buildenv\2.3-j00019-x64-release-fastbuild-static-55e94982-minimal\lib\libogg.lib" "C:\buildenv\2.3-j00019-x64-release-fastbuild-static-55e94982-minimal\lib\libvorbisfile.lib" 
```